OTKit/otkit-borders/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "otkit-borders",
+  "version": "1.0.0",
```

Comment:
The convention I found good, is to make a new package having `0.0.1` - once merged, we can publish as `1.0.0` with a `npm run publish` on the root - if you have 1.0.0 it forces you to bump it to something like 1.0.1 which is annoying

Reply:
Ok that makes sense. I will put this note in readme for people want to create new tokens in future.